aws/nlb/nlb.tf

```diff
@@ -0,0 +1,291 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# VARIABLES / LOCALS / REMOTE STATE
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "vpc_id" {
```

Comment:
The following descriptions are a bit misleading because the speak about `ALB` instead of `NLB`